wagtail/admin/forms/collections.py

```diff
@@ -145,13 +189,15 @@ def save(self):
         # get a set of (collection, permission) tuples for all ticked permissions
         forms_to_save = [
             form for form in self.forms
-            if form not in self.deleted_forms and 'collection' in form.cleaned_data
```

Comment:
For clarity, could we avoid changes to lines that aren't part of the functional changes on this PR? I think these may be due to black formatting being applied by mistake.

Reply:
I'll look into it.